src-ui/src/app/components/common/edit-dialog/edit-dialog.component.ts

```diff
@@ -2,104 +2,111 @@ import { Directive, EventEmitter, Input, OnInit, Output } from '@angular/core';
 
   @Input()
-  object: T
+  object: T;
 
   @Output()
-  success = new EventEmitter()
+  success = new EventEmitter();
 
-  networkActive = false
+  networkActive = false;
 
-  closeEnabled = false
+  closeEnabled = false;
 
-  error = null
+  error = null;
 
-  abstract getForm(): FormGroup
+  abstract getForm(): FormGroup;
 
-  objectForm: FormGroup = this.getForm()
+  objectForm: FormGroup = this.getForm();
 
   ngOnInit(): void {
     if (this.object != null) {
-      this.objectForm.patchValue(this.object)
+      this.objectForm.patchValue(this.object);
     }
 
     // wait to enable close button so it doesnt steal focus from input since its the first clickable element in the DOM
     setTimeout(() => {
-      this.closeEnabled = true
+      this.closeEnabled = true;
     });
   }
 
   getCreateTitle() {
-    return $localize`Create new item`
+    return $localize`Create new item`;
   }
 
   getEditTitle() {
-    return $localize`Edit item`
+    return $localize`Edit item`;
   }
 
   getSaveErrorMessage(error: string) {
-    return $localize`Could not save element: ${error}`
+    return $localize`Could not save element: ${error}`;
   }
 
   getTitle() {
     switch (this.dialogMode) {
       case 'create':
-        return this.getCreateTitle()
+        return this.getCreateTitle();
       case 'edit':
-        return this.getEditTitle()
+        return this.getEditTitle();
       default:
         break;
     }
   }
 
   getMatchingAlgorithms() {
-    return MATCHING_ALGORITHMS
+    return MATCHING_ALGORITHMS;
   }
 
   get patternRequired(): boolean {
-    return this.objectForm?.value.matching_algorithm !== MATCH_AUTO
+    return this.objectForm?.value.matching_algorithm !== MATCH_AUTO;
   }
 
   save() {
-    var newObject = Object.assign(Object.assign({}, this.object), this.objectForm.value)
-    var serverResponse: Observable<T>
+    var newObject = Object.assign(
+      Object.assign({}, this.object),
+      this.objectForm.value
+    );
+    var serverResponse: Observable<T>;
     switch (this.dialogMode) {
       case 'create':
-        serverResponse = this.service.create(newObject)
+        serverResponse = this.service.create(newObject);
         break;
       case 'edit':
-        serverResponse = this.service.update(newObject)
+        serverResponse = this.service.update(newObject);
       default:
         break;
     }
-    this.networkActive = true
-    serverResponse.subscribe(result => {
-      this.activeModal.close()
-      this.success.emit(result)
-    }, error => {
-      this.error = error.error
-      this.networkActive = false
-    })
+    this.networkActive = true;
+    serverResponse.subscribe(
+      (result) => {
+        this.activeModal.close();
+        this.success.emit(result);
+      },
+      (error) => {
+        this.error = error.error;
+        this.networkActive = false;
+      }
+    );
   }
 
   cancel() {
-    this.activeModal.close()
+    this.activeModal.close();
   }
 }
```

Comment:
There is no change, right? I would preffer to revert this changes so we don't mess up the git history?

Reply:
sure. my installed prettier from the IDe tried to solve some issues :-D